plugin-locationcomponent/src/main/res-public/values/public.xml

```diff
@@ -46,6 +46,10 @@
     <public name="mapbox_locationComponentLocationPuckLocationPuck3DModelScale_z" type="attr" />
     <!-- The scale expression of the model, which will overwrite the default scale expression that keeps the model size constant during zoom. -->
     <public name="mapbox_locationComponentLocationPuckLocationPuck3DModelScaleExpression" type="attr" />
+    <!-- The translation of the model [lon, lat, z] -->
```

Comment:
Is `lon, lat, z` naming aligned with gl-native? Sounds a bit strange to me

Reply:
Yes, that's copied from gl-native 😄